OrbitCore/OrbitModuleTest.cpp

```diff
@@ -0,0 +1,47 @@
+#include "Pdb.h"
+
+#include <gmock/gmock-matchers.h>
+#include <gtest/gtest.h>
+
+#include <utility>
+
+#include "Path.h"
+
+TEST(OrbitModule, LoadPdb) {
+  std::string executable_path = Path::GetExecutablePath();
+  std::string test_elf_file = executable_path + "/testdata/hello_world_elf";
```

Comment:
Where can I find more information about this binary? Is the source somewhere?

Reply:
This is a printf("Hello Earth!\n"); program.. recompiling it with different toolchain will likely break the test so I do not know if there is a value in adding source code for this to the repository. We definitely do not want to recompile it.

Reply:
I am going to have a similar situation when testing the build id things and testing whether a file contains symbols at all. I plan on doing the same thing, basically adding two more binaries here. (one elf file without symbols and a file containing only the symbols)

Reply:
Ok, I guess that in this case "hello_world_elf" gives it away... But in general I think that a comment about the test binary would be useful if one reads through the test later.

Reply:
added a comment